packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -19,18 +19,20 @@ describe('Reporter Header', () => {
     it('filters the list of specs when searching for specs', () => {
       cy.get('body').type('f')
 
-      cy.get('input').type('dom', { force: true })
+      cy.findByTestId('specs-list-panel').within(() => {
```

Comment:
Again more specific, we've got another `<input>` now for the Studio URL to visit for the test, so we need to make this more specific.